expression/builtin_compare_vec.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
+	"math"
```

Comment:
Please separate the standard library and the third-party library.

Reply:
done.